app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @recipes = Recipe.where(public: true).order(created_at: :desc).includes(:user)
+  end
```

Comment:
If a recipe is marked as public by the owner, any other user can mark it back as private. Any user is also able to add food items to a public recipe. That should not be the case. Kindly fix this issue.

Reply:
well resolved! ✔️